### ISSUE TITLE
Problem about the evaluation data

### ISSUE THREAD

**Issue by lucenzhong** [bepoetree/MTTOD#3]
https://github.com/bepoetree/MTTOD/blob/7256673b083448c8ac2fdd8bb95cdb9baf52b2b1/reader.py#L121
I found that in the original code, one test example(num_turn=18) will be discarded. Simply change the above code can fix this issue:
`if data_type != "test" and (k == 1 or k >= 17)`

**Comment by lucenzhong** [comment]
https://github.com/bepoetree/MTTOD/blob/7256673b083448c8ac2fdd8bb95cdb9baf52b2b1/evaluator.py#L136
https://github.com/bepoetree/MTTOD/blob/7256673b083448c8ac2fdd8bb95cdb9baf52b2b1/evaluator.py#L137
Here I found a new problem. When calculate the bleu score, the <bos_resp> and <eos_resp> token is not excluded from the response. Change the code to:
```
gen.append(" ".join(turn['resp_gen'].split()[1:-1]))
truth.append(" ".join(turn['resp_delex'].split()[1:-1]))
```

**Comment by bepoetree** [comment]
Hello, thank you for reporting the errors.

We have fixed them and found a little performance degradation (about 1 point in BLEU score).
To the best our knowledge, it is still state-of-the-art performance for end-to-end modelling.

We also have added conversion script to convert the format of result file for standardized evaluation of response generation.
We recommend using the standardized evaluation for fair comparison with other works.
Please refer to https://github.com/Tomiinek/MultiWOZ_Evaluation for details.

**Comment by lucenzhong** [comment]
Thanks for sharing standardized evaluation repository. Here Igot some new questions.

I wonder whether the hyper-paramenter setting in domain adaptation setting is the same as the full data training. And I read the Few-shot Domain Adaptation part in your paper,  found that Table 3 only contains the combined score in the source domain. Could you please release the combined score in the target domain as well? Thanks again.

**Comment by bepoetree** [comment]
In domain adaptation experiment, the hyper-parameter setting for training the rest of domains is same as the full data training.
However, **when we fine-tune the model on the target domain, the learning rate is set to 1e-3 and warmup_steps is set to 0.**
All combined scores for the rest of domains (except for target domain) are as follows:

| Model  | except. Train | except. Taxi | except. Restaurant | except. Hotel |
| ------------------ | ------------------ | ------------------ | ------------------ | ------------------ |
| MTTOD  | 101.53 | 104.09 | 94.75 | 102.07 |
| MTTOD w/o MTL  | 100.27  | 104.89 | 79.58 | 96.35 |

where 'except. Train' means Taxi + Restaurant + Hotel which is same as _Train - Others_ in our paper.

Note that these combined scores are result of fine-tuned model in the target domain.
